### DISCUSSION TITLE
Retreive field Last_x0020_Modified in Modern Search V4

### DISCUSSION THREAD

**Discussion by ragun101** [microsoft-search/pnp-modern-search#3587] [Q&A]
Hi everyone

I am working with SharePoint Document Sets in my environment and as you might know, the field `Modified` does not update, when an item inside a Document Set is updated, added or removed. However, the field `Last_x0020_Modified` is updated as per my tests with PnP PowerShell. Does anyone know if there is a way to get this CSOM properties into the search index or somehow use this field in PnP Modern Search? I tried to map the field onto a managed property but did not succeed. Any help would be appreciated. 😃 

Regards, ragun101

**Comment by wobba** [comment]
Last_x0020_Modified doesn't have a crawled property associated with it as it's set to be hidden, and hidden fields are not indexed.
